.github/workflows/release.yml

```diff
@@ -1,38 +1,35 @@
-name: Release
+name: Release to PyPI
```

Comment:
Not sure, but this formulation sounds strange to me. You release a version. And you publish this version on a platform (PyPI). But I wouldn't say that you release a version to a platform.

Reply:
good point. Changed it to publish.